htdocs/langs/fr_FR/products.lang

```diff
@@ -340,3 +340,4 @@ ErrorDestinationProductNotFound=Produit destination non trouvé
 ErrorProductCombinationNotFound=Variante du produit non trouvé
 ActionAvailableOnVariantProductOnly=Action disponible uniquement sur la variante du produit
 ProductsPricePerCustomer=Prix produit par clients
+GlobalSearchOnExtrafield=Recherche globale : Appliquer la recherche sur les extrafields (Attention : les dates sont au format YYYY-MM-DD)
```

Comment:
@atm-quentin 
add translation in en_US and after use transifex
and why (and where ?) the date format is "YYYY-MM-DD" ?

Reply:
in database

Reply:
@atm-quentin 

oui je me doute bien, mais pourquoi le préciser dans ta traduction ?
on doit rentrer la date dans ce format dans le champ de recherche ?
si oui pourquoi ne pas convertir la date dans ce format ?

Reply:
Oui,  je ne l'ai pas converti car c'est lourd.
C'est faisable, mais il faudrait mettre de coté tous les extrafields de type date, et leur appliquer un comportement particulier en fonction du pays dans lequel nous sommes dans la requete SQL. Ca me parait un peu lourd. 
Surtout qu'après réflexion si on veut pouvoir gérer tous les champs de types extrafields il faudrait pouvoir les gérer autrement (après la requete SQL en PHP par exemple) car les données sont stockés au format json et ça me parait compliqué de pouvoir les traiter directement dans la requête SQL.